game-guide/trading.md

```diff
@@ -1,54 +1,297 @@
+
+Some players will trade special routes or use tactics specifically to gain relations quickly and get it over with.  Trading at two adjacent ports will make more relations per turn than any longer route.  It's a fast way to make it to 500 personal relations.  Remember, after 500 each trade made will generate only 1 more point.  Traders will sometimes then switch to buying or selling one good at a time to finish their relations gain.  You lose some experience and credits by doing this but it gets you through the grind very quickly.
+
+Also take note that effective relations are personal plus political.  Your council may declare war but if you have 2000 personal relations it does not change what ports are open to you.  The opposite may be true if your personal relations are extremely low.  As a trader, you just stay focused on keeping your effective relations above -300 in order to keep ports open to you.  And keep grinding the ports you trade with to 1000 so you get full experience and the best profits.
+
+## Overhead Costs
+
+Federal protection costs no credits but you have to get there.  Overhead to a trader is a matter of turns.  How far do I have to go to get from safety to my route?  How far do I have to go to get back?  Do I have to go out of my way to get to the bank?
+
+While overhead values are usually small they will influence ship selection.  If you have to spend 20 turns to trade every day or two that is two hours worth of turns at 10tph or five hours at 4tph.  In that scenario you don't just spend less time getting there.  You use the saved time to trade and turn a profit.
+
+Those of you with stronger math skills will have a nagging idea of the slower ship taking longer to build turns, making less trips, and coming out equal.  The concept is correct but it clashes with the limitations of routes.  Namely supply.  There are very few scenarios where you can show up to a route and trade 500 turns out with 500 cargo holds.
+
+## Finding a Trade Route
+
+Finding a route is a matter of finding ports with good distance numbers.  It's an incredible amount of work to do it by hand so players have built applications to compute the routes for us. 
+
+Download your SMR file using the button to the lower left after you have the galaxy mapped.  Use The Lazy Trader to open the SMR file you downloaded and use the trade routes function to compute routes.  You will have to configure which goods you can trade depending on your alignment and which races you can trade with depending on your relations.
+
+The Lazy Trader can be found at [the game guide page for it](https://wiki.smrealms.de/tools/lazy-trader)
+
+## Trade Ships
+
+The essence of a trade ship is its ability to move goods.  It's not just how many fit in the hold at once.  It's how many can be transported.  This value can be referred to as *trade power*.  It can be measured by multiplying the number of cargo holds by the turn rate.  Calculate for many vessels to produce a chart like this:
```

Comment:
```suggestion
The efficiency of a trade ship depends on both its size (number of cargo holds) and its speed. For example: A slow ship with lots of cargo holds may be good, but a ship with fewer cargo holds may be better if it is much faster. This can be referred to as *trade power*.  It is measured by multiplying the number of cargo holds by the turn rate.  Calculate for many vessels to produce a chart like this:
```